frontend/src/components/Admin/AdminShoutouts/AdminShoutouts.tsx

```diff
@@ -131,8 +136,16 @@ const AdminShoutouts: React.FC = () => {
     if (view === 'PRESENT')
       return (
         <Item.Group divided>
+          <Header className={styles.presentCount}>
+            <Image className={styles.presentCountImg} src={catEmoji.src} alt="loading" />
```

Comment:
Not sure if I agree with the "loading" alt string. Maybe pick something that's closer to what this really is?